Stacey Brosnan/week02/Tuesday/css/bio.css

```diff
@@ -0,0 +1,91 @@
+@import url('https://fonts.googleapis.com/css2?family=Heebo:wght@300&family=Kufam&family=Teko:wght@300&display=swap');
+
+body {
+  background-color: #111;
+  padding:0;
+  margin:0;
+  font-family: 'Heebo', sans-serif;
+}
+
+.container {
+  max-width: 1100px;
+  background-color: lightblue;
+  margin: auto;
+}
+
+nav {
+  padding: 5px;
+  font-size: 30px;
+  text-align: center;
+  border-bottom: solid;
+  border-color: #222;
+}
+a:hover {
+  color: white;
+}
+
+
+a {
+  color: #808000;
+  text-decoration: none;
+  padding: 40px;
+  font-weight: bold;
+}
+
+
+h1{
+   text-align: center;
+   background-color: #222;
+   color: #808000;
+   font-size: 400%;
+   padding: 0.5em;
+   border-bottom-left-radius: .2em;
+   border-bottom-right-radius: .2em;
+   margin: auto;
+ }
+
+main{
+  padding-bottom: 50px;
+  padding-left: 50px;
+}
+
+h2{
+  color: #808000;
+  background-color: #222;
+  width: 250px;
+  border-radius: 0.1em;
+  text-align: center;
+  margin-top: 40px;
+
+}
+
+p{
+  text-align: left;
+  line-height: 1.5em;
+  padding-left: 30px;
+  padding-top: 10px;
+  padding-right: 30px;
+}
+
+.paragraphBorder1 {
+    box-sizing: border-box;
+    width: 900px;
+    height:100px;
+    box-shadow: 0 0 13px darkblue;
+    border-radius: 0.5em;
+}
+
+.paragraphBorder2 {
+    box-sizing: border-box;
+    width: 900px;
+    height:210px;
+    box-shadow: 0 0 13px darkblue;
+    border-radius: 0.5em;
+}
+.paragraphBorder3 {
+    box-sizing: border-box;
+    width: 900px;
+    height:70px;
+    box-shadow: 0 0 13px darkblue;
+    border-radius: 0.5em;
+}
```

Comment:
All of these paragraphBorder classes seem to have the same styling. You could re-write this as:

```
.paragraphBorder1, .paragraphBorder2, .paragraphBorder3 {
    box-sizing: border-box;
    width: 900px;
    box-shadow: 0 0 13px darkblue;
    border-radius: 0.5em;
}

.paragraphBorder1 {
    height:100px;
}

.paragraphBorder2 {
    height:210px;
}

.paragraphBorder3 {
    height:70px;
}
```